host/crypto/bcrypt/cert.c

```diff
@@ -28,9 +29,8 @@
 
 #define _OE_CERT_CHAIN_LENGTH_ANY 0
 
-static const DWORD _OE_DEFAULT_GET_CRL_FLAGS = CERT_STORE_SIGNATURE_FLAG |
-                                               CERT_STORE_TIME_VALIDITY_FLAG |
-                                               CERT_STORE_BASE_CRL_FLAG;
+static const DWORD _OE_DEFAULT_GET_CRL_FLAGS =
+    CERT_STORE_SIGNATURE_FLAG | CERT_STORE_TIME_VALIDITY_FLAG;
 
```

Comment:
I am curious what this flag was removed.

Reply:
I meant to remove CERT_STORE_TIME_VALIDITY_FLAG since CRL validity check is now done in revocation.c.  Also, the other cert libraries don't do this check.

Reply:
great!